locale/justoverclock-flarum-ext-realtimecode

```diff
@@ -0,0 +1,5 @@
+  forum:
+    title: HTML Editor
+    htmleditor: HTML Editor
+    description: Dieser HTML Editor unterstützt HTML/CSS code, Schreibe einfach deinen Code hier rein und schaue dir die Echtzeit-Vorschau an..
```

Comment:
HTML/CSS Code 
"schreibe" behind the comma lowercase, original tl doesn't include period at the end, please remove them.